### PR TITLE
Happy Blocks: Address navbar fixes

### PR DESCRIPTION
**Pull request by escapemanuele**
Related to pduY5I-CD-p2

This PR:
- Adds a trailing slash to /newsletter link
- Other links are now absolute

## Testing
1. Sync happy-blocks
2. Check that links in the navbar work in /learn

### CODE REVIEW

File: apps/happy-blocks/block-library/universal-header/index.php [hunk 1]
```diff
@@ -423,7 +423,7 @@ class="x-hidden"><?php esc_html_e( 'Close the navigation menu', 'happy-blocks' )
 							</li>
 							<li class="x-menu-grid-item">
 								<a role="menuitem" class="x-menu-link x-link"
-									href="<?php echo esc_url( localized_wpcom_url( '//wordpress.com/newsletter' ) ); ?>"
+									href="<?php echo esc_url( localized_wpcom_url( '//wordpress.com/newsletter/' ) ); ?>"
```
Comment (alshakero) [inline apps/happy-blocks/block-library/universal-header/index.php]: why no https here?

File: apps/happy-blocks/block-library/universal-header/index.php [hunk 2]
```diff
@@ -155,7 +155,7 @@ class="x-hidden"><?php esc_html_e( 'WordPress.com', 'happy-blocks' ); ?></span>
 						</li>
 						<li>
 							<a role="menuitem" class="x-dropdown-link x-link"
-								href="<?php echo esc_url( localized_wpcom_url( '//wordpress.com/newsletter' ) ); ?>"
+								href="<?php echo esc_url( localized_wpcom_url( '//wordpress.com/newsletter/' ) ); ?>"
```
Comment (alshakero) [inline apps/happy-blocks/block-library/universal-header/index.php]: +1

**Review by alshakero — APPROVED** [review]
Two notes.

File: apps/happy-blocks/block-library/universal-header/index.php [hunk 3]
```diff
@@ -211,7 +211,7 @@ class="x-icon x-icon--external" role="img" aria-hidden="true" focusable="false">
 					<ul role="menu">
 						<li>
 							<a role="menuitem" class="x-dropdown-link x-link"
-								href="<?php echo esc_url( localized_wpcom_url( '//wordpress.com/features/' ) ); ?>"
+								href="<?php echo esc_url( localized_wpcom_url( 'https://wordpress.com/features' ) ); ?>"
```
Comment (chickee0311) [inline apps/happy-blocks/block-library/universal-header/index.php]: Needs a trailing slash --> https://wordpress.com/features/

File: apps/happy-blocks/block-library/universal-header/index.php [hunk 4]
```diff
@@ -221,21 +221,21 @@ class="x-icon x-icon--external" role="img" aria-hidden="true" focusable="false">
...
-								href="<?php echo esc_url( localized_wpcom_url( '//wordpress.com/plugins' ) ); ?>"
+								href="<?php echo esc_url( localized_wpcom_url( 'https://wordpress.com/plugins' ) ); ?>"
 								tabindex="-1">
 								<?php echo esc_html( fixme__( 'WordPress Plugins', __( 'Plugins', 'happy-blocks' ) ) ); ?>
 							</a>
 						</li>
 						<li>
 							<a role="menuitem" class="x-dropdown-link x-link"
-								href="<?php echo esc_url( localized_wpcom_url( '//wordpress.com/google/' ) ); ?>"
+								href="<?php echo esc_url( localized_wpcom_url( 'https://wordpress.com/google' ) ); ?>"
```
Comment (chickee0311) [inline apps/happy-blocks/block-library/universal-header/index.php]: Needs a trailing slash --> https://wordpress.com/google/

File: apps/happy-blocks/block-library/universal-header/index.php [hunk 5]
```diff
@@ -478,7 +478,7 @@ class="x-icon x-icon--external" role="img" aria-hidden="true" focusable="false">
 						<ul class="x-menu-grid" role="menu">
 							<li class="x-menu-grid-item">
 								<a role="menuitem" class="x-menu-link x-link"
-									href="<?php echo esc_url( localized_wpcom_url( '//wordpress.com/features/' ) ); ?>"
+									href="<?php echo esc_url( localized_wpcom_url( 'https://wordpress.com/features' ) ); ?>"
```
Comment (chickee0311) [inline apps/happy-blocks/block-library/universal-header/index.php]: Needs a trailing slash --> https://wordpress.com/features/

File: apps/happy-blocks/block-library/universal-header/index.php [hunk 6]
```diff
@@ -487,21 +487,21 @@ class="x-icon x-icon--external" role="img" aria-hidden="true" focusable="false">
...
-									href="<?php echo esc_url( localized_wpcom_url( '//wordpress.com/plugins' ) ); ?>"
+									href="<?php echo esc_url( localized_wpcom_url( 'https://wordpress.com/plugins' ) ); ?>"
 									tabindex="-1">
 									<?php echo esc_html( fixme__( 'WordPress Plugins', __( 'Plugins', 'happy-blocks' ) ) ); ?>
 								</a>
 							</li>
 							<li class="x-menu-grid-item">
 								<a role="menuitem" class="x-menu-link x-link"
-									href="<?php echo esc_url( localized_wpcom_url( '//wordpress.com/google/' ) ); ?>"
+									href="<?php echo esc_url( localized_wpcom_url( 'https://wordpress.com/google' ) ); ?>"
```
Comment (chickee0311) [inline apps/happy-blocks/block-library/universal-header/index.php]: Needs a trailing slash --> https://wordpress.com/google/

**Pull request merged**